### PR TITLE
Mock travel pay claims in specs

### PR DESCRIPTION
**Pull request by athifw**
## Summary

We discovered that travel pay tests were hitting the travel pay claims endpoint on the dev instance. This PR mocks out any API calls in the travel pay tests.

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/82677

### CODE REVIEW

File: src/applications/travel-pay/tests/containers/TravelPayStatusApp.unit.spec.jsx [hunk 1]
```diff
@@ -38,6 +39,10 @@ describe('App', () => {
     global.window.location = {
       replace: sinon.spy(),
     };
+    const mockTravelClaims = {
+      data: [{ id: 'abc' }],
```
Comment (kjduensing) [inline src/applications/travel-pay/tests/containers/TravelPayStatusApp.unit.spec.jsx:43]: Can you update this to be an array of at least one "real" claim? This'll save Dan some work on the FE stuff and be more representative of what we'll get from the actual backend.

**Review by hinzed1127 — APPROVED** [review]
LGMT!

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by kjduensing — APPROVED** [review]
Thanks Athif! 🌵

**Pull request merged**